docs/pages/kubernetes-access/getting-started.mdx

```diff
@@ -1,204 +1,311 @@
 ---
-title: Connect a Kubernetes Cluster to Teleport
-description: Connecting a Kubernetes cluster to Teleport
+title: Enroll a Kubernetes Cluster
+description: Demonstrates how to enroll a Kubernetes cluster as a resource protected by Teleport.
 videoBanner: 3AUGrOZ5me0
 ---
 
-In this guide, we will show you how to register a Kubernetes cluster with
-Teleport by deploying the Teleport Kubernetes Service on the Kubernetes cluster
-you want to register.
+This guide demonstrates how to enroll a Kubernetes cluster as a resource by deploying the 
+Teleport Kubernetes Service on the Kubernetes cluster you want to protect. In this scenario, 
+the Teleport Kubernetes Service pod detects that it is running on Kubernetes and enrolls the 
+cluster automatically. The following diagram provides a simplified overview of this deployment
```

Comment:
```suggestion
Kubernetes cluster automatically. The following diagram provides a simplified overview of this deployment
```

Both Teleport and Kubernetes have the concept of a cluster. I think it's helpful to be clear about which one.